src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderBookingFilter.java

```diff
@@ -62,7 +62,6 @@ public Predicate toPredicate(@NotNull final Root<OffenderBooking> root, @NotNull
 
         Optional
             .ofNullable(caseloadIds)
-            .filter(java.util.function.Predicate.not(List::isEmpty))
```

Comment:
Removal of this line was needed to allow for a user with no caseloads.  Otherwise a user with no caseloads had the same result as an "override role" which meant everything was returned